flink-scala/pom.xml

```diff
@@ -236,4 +230,23 @@ under the License.
 		</plugins>
 	</build>
 
+	<profiles>
+		<profile>
+			<id>scala-2.10</id>
+			<activation>
+				<property>
+					<!-- this is the default scala profile -->
+					<name>!scala-2.11.profile</name>
```

Comment:
Why are you activating the profile via this property?
You can just do "mvn install -Pscala-2.10" to activate the profile.